pulpcore/app/serializers/status.py

```diff
@@ -72,6 +72,9 @@ class ContentSettingsSerializer(serializers.Serializer):
 
     content_origin = serializers.CharField(
         help_text=_("The CONTENT_ORIGIN setting for this Pulp instance"),
+        allow_blank=True,
```

Comment:
```suggestion
        allow_blank=False,
```

General advice from DRF: Don't allow two ways to say N/A.